src/libsyntax/ext/base.rs

```diff
@@ -636,7 +636,7 @@ pub struct ExpansionData {
 
 /// One of these is made during expansion and incrementally updated as we go;
 /// when a macro expansion occurs, the resulting nodes have the backtrace()
-/// -> expn_info of their expansion context stored into their span.
+/// -> `expn_info` of their expansion context stored into their span.
```

Comment:
The quote should include `->` as well, probably `` `backtrace() -> expn_info` ``